geoh5py/io/h5_reader.py

```diff
@@ -357,11 +345,17 @@ def fetch_value_map(cls, file: str | h5py.File, uid: uuid.UUID) -> dict | None:
         with fetch_h5_handle(file) as h5file:
             name = list(h5file.keys())[0]
             try:
-                values = np.r_[h5file[name]["Data"][cls.uuid_str(uid)]["Data"]]
+                entity = h5file[name]["Data"][cls.uuid_str(uid)]
```

Comment:
this would require its dedicated issue, PR and test, rather than being engulfed in a merge to main for a release